.gitignore

```diff
@@ -14,4 +14,4 @@ rustc-ice*.txt
 tests/native-lib/libtestlib.so
 .auto-*
 
-./genmc/
+genmc/
```

Comment:
This will ignore all folders called `genmc` everywhere, right? That doesn't sound right?